data/changelog.txt

```diff
@@ -1,3 +1,584 @@
+Expanded NPC conditions; spell levels, loaded mods & proficiencies
+Document emitters
+Document fields
+Redoes Ammo stats in Game_Balance
+
+## Mods:
+Use size flags for size changing mutations.
+Dinomod now has a rideable zombie T-Rex for all you Dresden fans
+New Dinomod evolutions
+Aftershock has some new robots and a new swarming monster AI behavior that can be used in all mods/mainline
+Magiclysm may have completed the attunement system or is significantly closer to completion
+Aftershock has new fauna monsters
+More atmospheric lore and details for Aftershock
+Dinomod has expanded it's dialogue options with mainline NPCs
+The DinoLab
+The Forge of Wonders now has three shops and stronger defenses on the vault
+The No Hope mod is available to greatly reduce the average player lifespan
+Dinomod specific achievements as well as Aftershock specific achievements
+Aftershock: Translocators for semi-convenient long range teleporting.
+Burned Zombie Dinos
+New gizmos and gadgets in Aftershock
+Aftershock: Stratoscomm Relay Station
+[Magiclysm] Adds new zombie animals
+Add mythos monsters to Mythos mod
+Dinomod Apex predators don't appear randomly day one
+Adds TropiCataclysm to the repository
+craftable holiday sweaters for dinos
+Hylian Adventurer Profession for Mythical Martial Arts
+Many south american animals added to Tropicataclysm
+Adds an innawood mod for wilderness survival.
+Aftershock Exoplanet: Glacial tunnels, a new low-risk area and the fauna that inhabits them
+TropiCataclysm Adds Termites and new content for Triffids
+Adds cast-bronze tools to Innawoods mod.
+[Innawood] Chemistry, electricity, and more!
+Many new dinosaurs added to Dinomod
+A mod to add subtitles to Rubik's dialogue in English.
+Tameable Wildlife MOD
+Dark Days of the Dead: Headshot requirements and deadly zombie bites are in!
+Dinomod NPC Missions
+New Mod Xedra Evolved with new magic classes, monsters, professions and CBMs plus Cyborg professions
+Desert region revival
+Add Megafauna Mod
+[DinoMod] tyrannosaurus bio-operator
+[DinoMod] Nanuqsaurus and burned rebuild
+Aftershock: Add functional jet packs
+Xedra Evolved: New NPCs to meet and rescue
+[DinoMod] player mutations change monster faction behavior
+[DinoMod] stegoceras
+Magiclysm Adding new magic plants
+[DinoMod] compsognathus, eoraptor, amargasaurus bio-operators
+Magiclysm Quest & Lore patch to item distribution
+[DinoMod] mi-go scout tower dino variant
+The Backrooms mod
+Vampire Virus in Xedra Evolved
+Crazy Cataclysm: Adds Powerthirst and Rocket Cans
+Xedra Evolved magic classes Eaters, Dreamers, Inventors and Dreamsmiths
+[Magiclysm] Make Summoning Great Again!
+Implementing more proper alchemy and some another stuff
+Aftershock Space items and smartguns
+Aftershock New CBM and class
+Aftershock: New Monofilament Whip CBM
+My Sweet Cataclysm: Butchering candy monsters gives candy flesh
+Add standard combat test mod
+Nested Crafting Categories
+Add cosmetic variants for mutations
```

Comment:
```suggestion
Aftershock
New robots and a new swarming monster AI behavior that can be used in all mods/mainline
New fauna monsters
More atmospheric lore and details
Achievements
Translocators for semi-convenient long range teleporting.
New gizmos and gadgets
Stratoscomm Relay Station
Add functional jet packs
Space items and smartguns
New CBM and class
New Monofilament Whip CBM
Aftershock Exoplanet: Glacial tunnels, a new low-risk area and the fauna that inhabits them

Dinomod
Rideable zombie T-Rex for all you Dresden fans
New evolutions
Expanded dialogue options with mainline NPCs
The DinoLab
Achievements
Burned zombie dinos
Apex predators don't appear randomly day one
Craftable holiday sweaters for dinos
41 new dinosaur species, more than doubled
NPC Missions
Tyrannosaurus, compsognathus, eoraptor, and amargasaurus bio-operators
Player mutations change monster faction behavior
Mi-go scout tower dino variant

Innawood
Mod for wilderness survival
Adds cast-bronze tools
Chemistry, electricity, and more!

Magiclysm
Completed the attunement system or significantly closer to completion
The Forge of Wonders now has three shops and stronger defenses on the vault
New zombie animals
Adding new magic plants
Quest & Lore patch to item distribution
Make Summoning Great Again!
Implementing more proper alchemy and some another stuff
Use size flags for size changing mutations.
Nested crafting categories

TropiCataclysm
Now available
Many south american animals
Termites and new content for Triffids

Xedra Evolved
New Mod with new magic classes, monsters, professions and CBMs plus Cyborg professions
New NPCs to meet and rescue
Vampire Virus
Magic classes Eaters, Dreamers, Inventors and Dreamsmiths

The Backrooms, replaces normal mapgen with endless rooms
Crazy Cataclysm: Adds Powerthirst and Rocket Cans
Dark Days of the Dead: Headshot requirements and deadly zombie bites are in!
Desert region revival, showcases regional_map_settings changes
Megafauna Mod: adds extinct megafauna of the late Pleistocene and early holocene
My Sweet Cataclysm: Butchering candy monsters gives candy flesh
Mythical Martial Arts: Hylian Adventurer Profession
Mythos Mod: Add mythos monsters
No Hope: Available to greatly reduce the average player lifespan
Standard combat test suite, a standard combat test mod
Tamable Wildlife for taming quiet/small wildlife
Translate Complex Dialogue, a mod to add subtitles to Rubik's dialogue in English.

```

I tried to reorganize with the big mods and then all the single line items, all alphabetized

Reply:
All changelog entries are listed in chronological order from oldest to newest, so I don't think we should be changing these around.